app/web/config/tailwind.config.js

```diff
@@ -35,9 +38,10 @@ module.exports = {
           200: '#DBDBEC',
         },
         'ch-blue': {
-          600: '#79B2F8',
+          700: '#08466F',
+          650: '#0958BA',
           500: '5098F1',
-          300: '#08466F'
+          400: '#79B2F8',
```

Comment:
Still others colors to fix, but fixed blues at least so that 100 => 900 is light to dark